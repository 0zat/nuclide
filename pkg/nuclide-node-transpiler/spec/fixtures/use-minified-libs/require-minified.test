const Rx = require('rxjs');
const {Observable} = require('rxjs');

assert(Rx.Observable === Observable);
