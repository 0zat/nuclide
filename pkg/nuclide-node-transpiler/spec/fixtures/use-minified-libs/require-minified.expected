var Rx = require('rxjs/bundles/Rx.umd.min.js');

var _require = require('rxjs/bundles/Rx.umd.min.js');

var Observable = _require.Observable;

assert(Rx.Observable === Observable);
